db.json

```diff
@@ -1633,7 +1684,7 @@
   "application/smil+xml": {
     "source": "iana",
     "compressible": true,
-    "extensions": ["smi","smil"]
+    "extensions": ["smi","smil","sml","kino"]
   },
```

Comment:
.sml correct per https://www.iana.org/assignments/media-types/application/smil , but .kino is not mentioned there

Reply:
https://gitlab.freedesktop.org/xdg/shared-mime-info/-/commit/aeec6ec3c98441db041e81274478c41dd9a38483 & https://bugs.freedesktop.org/show_bug.cgi?id=3141

This is about Linux only software discontinued in 2009 https://en.wikipedia.org/wiki/Kino_(software)

https://sourceforge.net/p/kino/code/HEAD/tree/trunk/smilutils/ and the manpage under that dont give any indication that .dino was a recommended extension.

However there are some hits on github for .kino being used for smil files https://github.com/search?q=path%3A*.kino&type=code

So this looks like a low value extension, but still valid.